RebelAllianceBank/Accounts/CardAccount.cs

```diff
@@ -35,36 +35,62 @@ public void AddToTransactionLog(Transaction newTransaction)
 
         public void ShowTransactionLog()
         {
+             _transactionsLog.Reverse();
+            Console.WriteLine("-------------------------------------------------------------------------------------");
+            Console.WriteLine($"Nuvarande saldo på konto: {this.Balance:N2} {AccountCurrency}");
+            Console.WriteLine("-------------------------------------------------------------------------------------");
             _transactionsLog.Reverse();
```

Comment:
remove one _transactionsLog.Reverse();, from each of the three classes. As you call it two times before the foreach loop and one time after. And it should only be one before and after, right?

Reply:
Great find! Fixed